scripts/zones/The_Sanctuary_of_ZiTah/Zone.lua

```diff
@@ -4,71 +4,58 @@
+require("scripts/globals/zone");
 
 -----------------------------------
 -- Chocobo Digging vars
 -----------------------------------
 local itemMap = {
-                    -- itemid, abundance, requirement
-                    { 688, 117, DIGREQ_NONE },
-                    { 17296, 150, DIGREQ_NONE },
-                    { 880, 100, DIGREQ_NONE },
-                    { 833, 83, DIGREQ_NONE },
-                    { 696, 100, DIGREQ_NONE },
-                    { 690, 33, DIGREQ_NONE },
-                    { 772, 17, DIGREQ_NONE },
-                    { 773, 33, DIGREQ_NONE },
-                    { 4386, 9, DIGREQ_NONE },
-                    { 703, 7, DIGREQ_NONE },
-                    { 4096, 100, DIGREQ_NONE },  -- all crystals
-                    { 1255, 10, DIGREQ_NONE }, -- all ores
-                    { 656, 117, DIGREQ_BURROW },
-                    { 750, 133, DIGREQ_BURROW },
-                    { 749, 117, DIGREQ_BURROW },
-                    { 748, 8, DIGREQ_BURROW },
-                    { 751, 14, DIGREQ_BURROW },
-                    { 720, 1, DIGREQ_BURROW },
-                    { 699, 9, DIGREQ_BORE },
-                    { 720, 1, DIGREQ_BORE },
-                    { 4570, 10, DIGREQ_MODIFIER },
-                    { 4487, 11, DIGREQ_MODIFIER },
-                    { 4409, 12, DIGREQ_MODIFIER },
-                    { 1188, 10, DIGREQ_MODIFIER },
-                    { 4532, 12, DIGREQ_MODIFIER },
-                };
-
+    -- itemid, abundance, requirement
+    { 688, 117, DIGREQ_NONE },
+    { 17296, 150, DIGREQ_NONE },
+    { 880, 100, DIGREQ_NONE },
+    { 833, 83, DIGREQ_NONE },
+    { 696, 100, DIGREQ_NONE },
+    { 690, 33, DIGREQ_NONE },
+    { 772, 17, DIGREQ_NONE },
+    { 773, 33, DIGREQ_NONE },
+    { 4386, 9, DIGREQ_NONE },
+    { 703, 7, DIGREQ_NONE },
+    { 4096, 100, DIGREQ_NONE },  -- all crystals
+    { 1255, 10, DIGREQ_NONE }, -- all ores
+    { 656, 117, DIGREQ_BURROW },
+    { 750, 133, DIGREQ_BURROW },
+    { 749, 117, DIGREQ_BURROW },
+    { 748, 8, DIGREQ_BURROW },
+    { 751, 14, DIGREQ_BURROW },
+    { 720, 1, DIGREQ_BURROW },
+    { 699, 9, DIGREQ_BORE },
+    { 720, 1, DIGREQ_BORE },
+    { 4570, 10, DIGREQ_MODIFIER },
+    { 4487, 11, DIGREQ_MODIFIER },
+    { 4409, 12, DIGREQ_MODIFIER },
+    { 1188, 10, DIGREQ_MODIFIER },
+    { 4532, 12, DIGREQ_MODIFIER },
+};
 local messageArray = { DIG_THROW_AWAY, FIND_NOTHING, ITEM_OBTAINED };
 
------------------------------------
--- onChocoboDig
------------------------------------
 function onChocoboDig(player, precheck)
     return chocoboDig(player, itemMap, precheck, messageArray);
 end;
 
------------------------------------
--- onInitialize
------------------------------------
-
 function onInitialize(zone)
-    local Noble_Mold = 17273278;
-    GetMobByID(Noble_Mold):setLocalVar("ToD",os.time() + math.random((43200), (57600)));
+    GetMobByID(NOBLE_MOLD):setLocalVar("pop",os.time() + math.random(43200, 57600)); -- 12 to 16 hr
```

Comment:
👍 death to extra parenthesis
This message approved by the Council for Filling Whasf's Inbox and the Formatting Fresh Sanity (FFS) group.